### PR TITLE
Added Web-Programming (Open Weather Maps data fetch)

### PR DESCRIPTION
**Pull request by ruppysuppy**
# Welcome to JavaScript community

### **Describe your change:**

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new JavaScript files are placed inside an existing directory.
* [X] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [X] Fixes: #187

### CODE REVIEW

File: Web-Programming/OpenWeatherMaps.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const fetch = require('node-fetch')
```
Comment (itsvinayak) [inline Web-Programming/OpenWeatherMaps.js:1]: what about this requirement, 
we can add a common package.json file which can be used to install all requirements using
`npm install`

and please remove **package-lock.json** file
thank you @ruppysuppy

Comment (itsvinayak) [reply Web-Programming/OpenWeatherMaps.js:1]: and please add a global package.json file

Comment (ruppysuppy) [reply Web-Programming/OpenWeatherMaps.js:1]: I have added a global package.json file and removed package-lock.json but the HeapSort was causing some issues, I didn't touch it but it was modified by itself.

**Comment by itsvinayak** [comment]
nice work !!

**Comment by ruppysuppy** [comment]
Thanks a lot for helping me out with the branch issues

**Pull request merged**